addons/binding/org.openhab.binding.roomba/README.md

```diff
@@ -0,0 +1,39 @@
+# Roomba Binding
+
+The Roomba binding is used to enable communication between openHAB and iRobot's Roomba 900 series where WiFi is available. This binding enables you to clean, dock, pause, and resume using openHAB.
+
+## Devices Supported
+
+Currently supported devices include:
+* iRobot Roomba 960
+* iRobot Roomba 980
+
+## Device Discovery
+
+The Roomba 900 series can be discovered using a broadcast UDP Packet. This binding will broadcast this message to the entire LAN and as long as the Roomba is connected to the WiFi network, the binding will autodiscover the Roomba and add it to your inbox.
+
+## Password
+
+In order to finalize setup, you will need to obtain the password to your Roomba. Instructions are available on the [https://github.com/koalazak/dorita980](dorita980) project.
+
+## Channels
+
+TV's support the following channels:
+
+| Channel Type ID | Item Type    | Description  |
```

Comment:
http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table 👍